### ISSUE TITLE
Support :focus for rspec permissions blocks

### ISSUE THREAD

**Issue by pas256** [varvet/pundit#816]
When writing tests using rspec, there is a nice helper called `permissions` that works similar to `describe`, but it does not support the `:focus` attribute used to limit which tests are run.

Example:

```ruby
RSpec.describe SheetPolicy, type: :policy do
  permissions :index? do
    it 'grants access if the workbook is created by the user' do
      expect(SheetPolicy).to permit(user, create(:sheet, workbook:))
    end
  end

  permissions :show?, :focus do
    it 'grants access if the workbook is created by the user' do
      expect(SheetPolicy).to permit(user, create(:sheet, workbook:))
    end
  end
```

When you have `config.filter_run_when_matching :focus` in your `spec_helper.rb` file, it should only run the `index` block of tests, but from what I can tell, `permissions` does not support it.

**Comment by Burgestrand** [comment]
Without looking into the details too much here, this feels something we should reasonably support.

**Comment by Burgestrand** [comment]
This was fixed in #820!
